### ISSUE TITLE
CSV Export attribute string missing "+" symbol

### ISSUE THREAD

**Issue by ntindicator** [woocommerce/woocommerce#21457]
**Describe the bug**
Using paper size variations for a product "A4, A3, A3+, A2". When the variations are exported there are 2 that are A3, and 1 should be A3+. Name field is also incorrect for the variations. This isn't used in the product, but it's also missing the "+" sign. 

**To Reproduce**
Create product with a global attribute, using the 4 sizes listed above. Export the CSV with the attributes. 

**Screenshots**
<img width="469" alt="variations" src="https://user-images.githubusercontent.com/30362690/46125558-08d1ef80-c26d-11e8-86a8-a7c115c7ed3e.png">
<img width="285" alt="export" src="https://user-images.githubusercontent.com/30362690/46125559-08d1ef80-c26d-11e8-94cd-bb24b1800d10.png">

**Expected behavior**
Exported CSV should show "A3+" in "Attribute 1 value(s)" field

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by 79mplus-admin** [comment]
Hello there,

I am wondering if we do need a fix for this since '+' sign has been removed during data validation for security reasons? Do we still consider this as a bug?

Looking forward to hearing from you soon.

Cheers.

**Comment by claudiulodro** [comment]
I'll have to look into it some more. All that the security check does is wrap the value in `''` single quotes to prevent Excel from thinking it's a script; it doesn't remove the `+`.

**Comment by rrennick** [comment]
https://github.com/woocommerce/woocommerce/blob/master/includes/export/abstract-wc-csv-exporter.php#L382 converts the '+' into a ' '. 

When that line is removed the string isn't quoted. A way that could be fixed is compare the original string to the url decoded one & if they are different quote the string. The importer would need to be check for reversing the encoding.

@claudiulodro If that sounds like a good solution I can do a PR in the morning.

**Comment by claudiulodro** [comment]
Fixed in #21864
